File: apps/android/app/src/main/java/com/smartble/core/model/LogEntry.kt
package com.smartble.core.model

/**
 * 日志类型
 */
enum class LogType {
    Info,
    Success,
    Warning,
    Error,
    Receive,
    Send,
}

/**
 * 日志条目
 */
data class LogEntry(
    val message: String,
    val type: LogType,
    val timestamp: String = ""
)

File: apps/flutter/lib/core/models/log_entry.dart
/// 日志类型
enum LogType {
  info,
  success,
  warning,
  error,
  receive,
  send,
}

/// 日志条目
class LogEntry {
  final String message;
  final LogType type;
  final DateTime timestamp;

  LogEntry({
    required this.message,
    required this.type,
    required this.timestamp,
  });
}
